cllc-public-app/ClientApp/src/app/components/applications/application/application.component.html

```diff
@@ -260,20 +260,17 @@ <h3 class="blue-header">Rural Agency Store Details</h3>
                 <p>Please provide the following details about your Rural Agency Store:</p>
                 <!-- RAS Certificate Number -->
                 <app-field label="Rural Agency Store Certificate Number:" errorMessage="Please enter a certificate number">
-                  <input type="text" class="form-control" style="width: 200px;" id="certNumber" formControlName="description1">
+                  <input type="text" class="form-control" style="width: 200px;" id="certNumber" formControlName="description1" mask="000">
                 </app-field>
               
                 <section style="width:700px;">
-                  <app-field [valid]="isValidOrNotTouched('isOwner')" [showChevrons]="false"
-                    errorMessage="Please confirm that the current zoning permits manufacturing on the manufacturing site"
-                    style="margin-left:10px; background-color: blue;">
+                  <div>
                     <mat-checkbox formControlName="isOwner">
-                      This information is being submited by the owner of the business in respect of which the licence is to be issued
-                      or will become<br>the owner before the licence is issued.</mat-checkbox>
-                  </app-field>
-                  <app-field [valid]="isValidOrNotTouched('hasValidInterest')" [showChevrons]="false"
-                    errorMessage="Please confirm that the current zoning permits retail sales on the manufacturing site"
```

Comment:
If we want these error messages to show up we can look at implementing them a different way but these app fields screw up the checkbox html structure